.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+cache: pip
+python:
+  - "2.7"
+services:
+  - redis-server
+addons:
+  postgresql: "9.3"
+install:
+  - pip install -r pip-freeze.txt
+  - pip install coveralls
+  - npm install less
+  - npm install coffee-script
+before_script:
+  - psql -U postgres -c "CREATE USER temba WITH PASSWORD 'temba';"
+  - psql -U postgres -c "ALTER ROLE temba WITH SUPERUSER;"
+  - psql -d 'template1' -U postgres -c "CREATE EXTENSION postgis;"
+  - psql -d 'template1' -U postgres -c "CREATE EXTENSION postgis_topology;"
+  - psql -d 'template1' -U postgres -c "CREATE EXTENSION hstore;"
+  - psql -U temba postgres -c "CREATE DATABASE temba;"
+  - ln -s $TRAVIS_BUILD_DIR/temba/settings.py.dev $TRAVIS_BUILD_DIR/temba/settings.py
+script:
+  - coverage run --source="." manage.py test temba/api temba/assets temba/campaigns temba/channels temba/contacts temba/flows temba/ivr temba/locations temba/msgs temba/orgs temba/public temba/reports temba/schedules temba/triggers temba/utils temba/values --verbosity=2 --noinput && coverage report -m --include="temba/api/*,temba/assets/*,temba/campaigns/*,temba/channels/*,temba/contacts/*,temba/flows/*,temba/ivr/*,temba/locations/*,temba/msgs/*,temba/orgs/*,temba/public/*,temba/reports/*,temba/schedules/*,temba/triggers/*,temba/utils/*,temba/values/*" --omit="*/migrations/*,*/tests" -i
```

Comment:
There's an `after_success` config for travis that we can use here instead of the `&&`, which would allow us to split this line up. http://docs.travis-ci.com/user/build-configuration/#Build-Lifecycle